java/IceGrid/greeter/greeter-hall-with-replication.xml

```diff
@@ -0,0 +1,43 @@
+<icegrid>
+  <!-- An application describes nodes and servers deployed on these nodes. An IceGrid deployment (a registry plus one or
+  more nodes) can host several applications. -->
+  <application name="GreeterHall">
+
+    <!-- A server template allows us to instantiate several similar servers. -->
+    <server-template id="GreetServer">
+      <!-- This template has a single parameter, index. Its value can be any string. -->
+      <parameter name="index"/>
+
+      <!-- An instance of this template is a server with ID GreetServer-${index}. IceGrid starts this server when a
+      client looks up an object adapter or a well-known object in this server or in a replica group it references. -->
+      <server id="GreetServer-${index}" exe="java" activation="on-demand">
+        <option>--class-path</option>
+        <option>server/build/install/server/lib/*</option>
```

Comment:
Is the `*` necessary? Looks unusual for a CLASSPATH.

Reply:
Added comments to the descriptor options.